src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -97,6 +96,57 @@ public int compareTo(Code other) {
     }
   }
 
+  /**
+   * HealthRecord.ValueSet represents a valueset URL and display name.
+   */
+  public static class ValueSet implements Comparable<ValueSet> {
+    /** The ValueSet URL. */
+    public String url;
+    /** The human-readable description of the ValueSet. */
+    public String display;
```

Comment:
NIT: Do these need to be public?

Reply:
Currently yes; we're getting the URL directly from that variable in `Module::getValueSetUrls`

Reply:
Thoughts on being more Java-esque and writing a getter `getUrl` and `getDisplay` and changing these to private?

Reply:
I can, I'm just not sure how much that really buys us; plus that actually breaks from the convention set in the `HealthRecord.code` object I ~stole copied~ borrowed this code from.

Reply:
ahhhh dang didn't look at what else was used. As long as it's consistent then we good.